### DISCUSSION TITLE
snaps unable to run

### DISCUSSION THREAD

**Discussion by NikkiSatmaka** [SpiralLinux/SpiralLinux-project#95] [Q&A] [answered]
Hi,

I'm having trouble running snap packages on SpiralLinux Gnome.
Two packages I tried to run were authy and tradingview
Both returned `/user.slice/user-1000.slice/session-2.scope is not a snap cgroup`

Does anybody what caused this and how to fix it?

**Comment by geckolinux** [comment, marked as answer]
Hi there, sorry for the slow reply!

Could you please test this solution?
https://forum.snapcraft.io/t/cannot-launch-snap-applications-with-cgroup-v2/27700/6

**Reply by NikkiSatmaka** [reply]
It works! But we need to restart afterwards. Simply starting the service does't work. Thank you

**Reply by geckolinux** [reply]
Thanks for confirming. I think we'll leave this as a documentation issue unless it affects other user-installed software apart from Snaps.
